### ISSUE TITLE
GraphQL query area height does not resize since v1.20.x

### ISSUE THREAD

**Issue by simeon-walker** [usebruno/bruno#2605]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

The query pane of a GraphQL request now has a fixed height, 14 lines in my case. It does not resize to fit the window height.

I have tried the latest flatpak and appimage. Reverting to v1.19 does not have the issue.

### .bru file to reproduce the bug

A new collection with an empty GraphQL request can show the bug.

### Screenshots/Live demo link

![image](https://github.com/usebruno/bruno/assets/1379208/de6cb273-7fe4-415e-80ad-a7460ec3ec8b)

**Comment by remino** [comment]
Getting the same problem. Looks like someone already made a fix for this way back but it never got merged:
https://github.com/usebruno/bruno/pull/912

**Comment by simeon-walker** [comment]
> Getting the same problem. Looks like someone already made a fix for this way back but it never got merged: #912

I see this issue too but I don't think it's the same thing, would be nice if it were merged though.

**Comment by kimnilsson** [comment]
Bug is still there in `1.22.0`. Downgrading to `1.19.0` as a workaround works.

**Comment by kimnilsson** [comment]
This issue was introduced in this [commit](https://github.com/usebruno/bruno/commit/0d3765ad66c11ed047deda1b0d8b26ae83235547).  CSS isn't really my cup of tea so I'm not sure how to fix it, but there's a quite suspicious [todo](https://github.com/usebruno/bruno/commit/0d3765ad66c11ed047deda1b0d8b26ae83235547#diff-52299d930fd51be8e9015d1a919806af827ee6da134ae91127e25601acfba65bL7) in there.

**Comment by kimnilsson** [comment]
Adding the following style attributes to `div.CodeMirror` seems to solve it (for me at least):

```css
@@ -4,6 +4,8 @@ const StyledWrapper = styled.div`
   div.CodeMirror {
     background: ${(props) => props.theme.codemirror.bg};
     border: solid 1px ${(props) => props.theme.codemirror.border};
+    min-height: 0px;
+    flex: 1 1 0;
   }
```

However I don't really understand the changes so wont make a PR - but perhaps someone else actually does :)

**Comment by helloanoop** [comment]
Thanks for sharing the `flex: 1 1 0` approach @kimnilsson !

This now has been fixed in the PR #2758
This will be released in `v1.24.0` which will be released tomorrow.
